extension/src/plots/webview/messages.ts

```diff
@@ -411,6 +414,7 @@ export class WebviewMessages {
     if (webview) {
       return {
         ...plot,
+        resourceUrl: plot.url,
```

Comment:
[Q] Can we encode/decode the url instead of duplicating? 

Note: might not be worth the effort.

Reply:
the only place we would have to decode would be directly before `openImageFileInEditor`.

Reply:
I'lll try something, but I know I hard a really hard time trying to decode it, that's why I decided it might be simpler to just pass a new value, but with the amount of changes in the tests, it might be worth trying again.